steps_file.js

```diff
@@ -4,10 +4,15 @@ module.exports = function() {
   return actor({
     openStore(){
       this.amOnPage('http://automationpractice.com/index.php');
-    }
+    },
+
+    openProduct(){
```

Comment:
use `I.amOnPage()` in scenario instead of this method, since it is specific for one product only

Reply:
Done